<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reword2</class>
 <widget class="QWidget" name="reword2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1171</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1171</width>
    <height>690</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>690</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1171</width>
     <height>690</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1171</width>
     <height>690</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>492</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_nouse">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_1">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_2">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_3">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_4">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_5">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_6">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_7">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_8">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_9">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wordmeaning_10">
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>602</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_newwordpage">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>生词本</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_1">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_2">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_3">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_4">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_5">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_6">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_7">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_8">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_9">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_word_10">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>0</y>
      <width>95</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <spacer name="verticalSpacer_10">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_last">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>上
一
页</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_next">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>下
一
页</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_return">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返

回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
